packages/bitcoin/src/fungible.ts

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+type BitcoinTransferRequest = {
```

Comment:
@MakMuftic @mpetrunic here I think we could be missing the taproot address to where the deposits goes. If we are going to handle this through shared config, do we want to provide this as explicit parameter or do we want to handle this behind the scenes in the sdk according to the env and the route?

Reply:
It should be behind the scenes